pywwise/ak/wwise/core/transport.py

```diff
@@ -1,54 +1,154 @@
-		play, stop, pause and resume the Wwise object via the other transport functions.
-		"""
-	
-	def destroy(self):
-		"""
-		Destroys the given transport object.
-		"""
-	
-	def execute_action(self):
-		"""
-		Executes an action on the given transport object, or all transport objects if none is specified.
-		"""
-	
-	def get_list(self):
-		"""
-		Returns the list of transport objects.
-		"""
-	
-	def get_state(self):
-		"""
-		Gets the state of the given transport object.
-		"""
-	
-	def prepare(self):
-		"""
-		Prepare the object and its dependencies for playback. Use this function before calling
-		`PostEventSync` or `PostMIDIOnEventSync` from `IAkGlobalPluginContext`.
-		"""
-	
-	def use_originals(self):
-		"""
-		Sets the Original/Converted transport toggle globally. This allows playing the original or the
-		converted sound files.
-		"""
+    """ak.wwise.core.transport"""
+    
+    def __init__(self, client: _WaapiClient):
+        """
+        Constructor.
+        :param client: The WAAPI client to use.
+        """
+        self._client = client
+        
+        # TODO: implement topics
+        self.state_changed: _RefEvent
+    
+    def create(self, wwise_object: tuple[EObjectType, Name] | GUID | ProjectPath,
+               game_object: GameObjectID = None) -> int:
+        """
+        https://www.audiokinetic.com/en/library/edge/?source=SDK&id=ak_wwise_core_transport_create.html \n
+        Creates a transport object for the given Wwise object. The return transport object can be used to
+        play, stop, pause and resume the Wwise object via the other transport functions.
+        :arg wwise_object: The ID (GUID), name, or project path of the object to control via the transport object.
+        :arg game_object: The game object to use for playback.
+        :return: Transport object ID to be used with all other ak.wwise.core.transport functions. Returns -1 if function
+        call failed.
+        """
+        if wwise_object is None:
+            return -1
+        
+        args = {"object": wwise_object if not
+                isinstance(wwise_object, tuple) else f"{wwise_object[0]}_{wwise_object[1]}"}
+        
+        if game_object is not None:
+            args["gameObject"] = game_object
+        
+        result = self._client.call("ak.wwise.core.transport.create", args)
+        
+        return -1 if result is None else result.get("transport", -1)
+    
+    def destroy(self, transport_id: int) -> bool:
+        """
+        https://www.audiokinetic.com/en/library/edge/?source=SDK&id=ak_wwise_core_transport_destroy.html \n
+        Destroys the given transport object.
+        :param transport_id: The transport object ID to be used with all other ak.wwise.core.transport functions. Make
+        sure you create a transport object before calling this function.
+        :return: True if the transport object was destroyed, False otherwise.
+        """
+        if transport_id is None:
```

Comment:
Check not needed in this context. I'd recommend removing it. In type-hinted Python code, if a parameter is allowed to have `None` as a value, we usually add `NoneType` to its typehint.